docs/guides/primitive-input-output.ipynb

```diff
@@ -0,0 +1,461 @@
+{
```

Comment:
<p>In this example, readers might wonder why does (100,2)x(3,1) turn into (3, 100) (where does the 2 go?). It's explained above, but it probably doesn't hurt to repeat:</p><p> </p>

<pre class="ql-syntax" spellcheck="false"># This shape is due to our array of parameter bindings having shape
# (100, 2), combined with our array of observables having shape (3, 1), where 2 is the number of parameters in the circuit.
</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Qiskit/documentation/pull/1621/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Reply:
<p>:+1: and if we want to be even more explicit <code>...where 2 is the number of parameters in the circuit and so does not count as being part of the shape of parameter _sets_.</code> </p><p> </p><p> </p><p>but, sometimes less is more, so at your discretion.</p>